include/picongpu/plugins/ResourceLog.hpp

```diff
@@ -239,7 +196,8 @@ namespace picongpu
                     std::stringstream ss;
                     ss << outputFilePrefix << rank;
                     boost::filesystem::path resourceLogPath(ss.str());
-                    fileBuf.open(resourceLogPath.string().c_str(), std::ios::out);
```

Comment:
Note: There was no need to change this because the pointer from `c_str()` life as long as the expression is executed.
The construct is only wrong if you write:

```
auto tmp = resourceLogPath.string().c_str();
fileBuf.open(tmp, std::ios::out);
```

Reply:
That's true.